botlistbot/botcheckerworker/botchecker.py

```diff
@@ -13,7 +13,7 @@
 import traceback
 from datetime import datetime, timedelta
 from logzero import logger as log
-from pyrogram.api.functions.contacts import Search
+from pyrogram.api.functions.contacts from botlistbot import search
```

Comment:
Looks like a typo :thinking:

Reply:
Hehe, regex fails

Reply:
Good thing I was wise enough not to commit to master 😆